### PR TITLE
Edits and updates for Moderation page

### PR DESCRIPTION
**Pull request by ghost**
Added new section on moderator expectations from https://beehaw.org/post/468998

### CODE REVIEW

File: content/docs/core-functions/moderation.md [hunk 1]
```diff
@@ -3,27 +3,47 @@ title: Moderation
...
+We think in the long term moderation is likely the most important aspect of this instance and key to ensuring this community stays a nice place. Moderation is a tricky subject and something that we don’t think any platform on the internet has managed to figure out.
 
-Rather than try to summarize about our current philosophy on moderation we’d like to focus on some issues we’ve seen (some of which we may not have an answer to yet).
+Rather than try to summarize our current philosophy on moderation we’d like to focus on some issues we’ve seen (some of which we may not have an answer to yet).
 
-As many of you new folks are refugees from Reddit, some of the problems I’m about to outline are likely problems you’ve seen with moderation. 
+As many Beehaw folks are refugees from Reddit, some of the problems we're about to outline are likely problems you’ve seen with moderation.
```
Comment (lionirdeadman) [inline content/docs/core-functions/moderation.md]: Beeple

File: content/docs/core-functions/moderation.md [hunk 2]
```diff
@@ -3,27 +3,47 @@ title: Moderation
...
 
 ### Elections
-Often people like to point to elections as a way to self-govern. Without even naming some of the issues with elections, we're sure most of you can look at existing elected governmental officials in nearly any country and easily identify at least one person who highlights issues with this process. By its very nature election often becomes a popularity contest, rife for abuse in a plethora of ways that humans with good social skills often use to their advantage.
+Often people like to point to elections as a way to self-govern. Without even naming some of the issues with elections, we're sure most of you can look at existing elected governmental officials in nearly any country and easily identify at least one person who highlights issues with this process. By its very nature an election often becomes a popularity contest, rife for abuse in a plethora of ways that humans with good social skills often use to their advantage.
```
Comment (lionirdeadman) [inline content/docs/core-functions/moderation.md]: By its very nature should have a comma after it

File: content/docs/core-functions/moderation.md [hunk 3]
```diff
@@ -3,27 +3,47 @@ title: Moderation
...
+Sortition (appointment by lot) may offset some of these problems but also has its pros and cons. Perhaps most importantly, we don’t want to burden anyone who’s not interested in moderating with that responsibility. We think people would be well served to examine the social groups which exist in their lives which aren’t governed via direct democracy and to spend a bit of time considering what model is most appropriate in each sphere of their life.
 
 ### Legitimacy
 We wouldn't want to receive medical care from an individual which was elected by the vote of non-medical professionals. The same would be true of legal advice. We would like our moderators to be educated and skilled in moderating, and both elections and sortition don’t always center these values. Ultimately, we haven’t decided on a sustainable long-term solution to moderation, and have been choosing active members in communities which seem to embody or align with our ethos to elevate to a moderating position.
+
+## Moderation at Beehaw
```
Comment (lionirdeadman) [inline content/docs/core-functions/moderation.md:31]: Will ask if we wanna include this in docs

**Review by lionirdeadman — CHANGES_REQUESTED** [review]

File: content/docs/core-functions/moderation.md (hunk 1 above)
Comment (ghost) [reply content/docs/core-functions/moderation.md]: Fixed. Also, oops.

File: content/docs/core-functions/moderation.md (hunk 2 above)
Comment (ghost) [reply content/docs/core-functions/moderation.md]: Fixed

File: content/docs/core-functions/moderation.md (hunk 3 above)
Comment (ghost) [reply content/docs/core-functions/moderation.md:31]: Let me know if you want me to strip it back out.

**Pull request merged**